tools/SetupFlow/DevHome.SetupFlow/Services/ConfigurationFileBuilder.cs

```diff
@@ -180,22 +184,22 @@ private WinGetConfigResource CreateResourceFromTaskForWinGetDsc(InstallPackageTa
         {
             // WinGet configure uses the Id property to uniquely identify a resource and also to display the resource status in the UI.
             // So we add a description to the Id to make it more readable in the UI. These do not need to be localized.
-            id = $"{arguments.PackageId} | Install: " + task.PackageName;
+            id = $"{arguments.PackageId}{PackageNameSeparator}{task.PackageName}";
```

Comment:
Just wanted to double check if this works.
The line `dependsOn : Git.Git | Install: Git` is actually causing tasks to fail for Dev Boxes.

Reply:
I am not sure what the question is. The change in this line is only replacing "| Install:" with a constant to use it in parsing when we receive results. There is no change in functionality. And yes, it works for local and Hyper-V setup. In both cases we send unchanged YAML file we generate here to WinGet. If DevBox doesn't work we'll need to understand if they do anything with the YAML before sending to WinGet.

Reply:
Yeah, I don't mean this line in general but the whole dependsOn. FYI - It doesn't work for DevBox (and as a work around we will remove it during parsing.)